2019-September.html

```diff
@@ -0,0 +1,251 @@
+
+        <!-- For syntax highlighting -->
+        <link rel="stylesheet" href="lib/css/zenburn.css">
+
+        <!-- Lobster font -->
+        <link href='https://fonts.googleapis.com/css?family=Lobster' rel='stylesheet' type='text/css'>
+
+        <!-- Font awesome for icons -->
+        <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
+
+        <!-- If the query includes 'print-pdf', include the PDF print sheet -->
+        <script>
+            if( window.location.search.match( /print-pdf/gi ) ) {
+                var link = document.createElement( 'link' );
+                link.rel = 'stylesheet';
+                link.type = 'text/css';
+                link.href = 'css/print/pdf.css';
+                document.getElementsByTagName( 'head' )[0].appendChild( link );
+            }
+        </script>
+
+        <!--[if lt IE 9]>
+        <script src="lib/js/html5shiv.js"></script>
+        <![endif]-->
+
+        <script>
+            slideData = {
+                "text": {
+                    "date": "September 2019",
+                    "speaker-name": "Bob den Otter",
+                    "joind-in": "https://joind.in/talk/06a20",
+                    "next-meetup-date": "17th of October",
+                    "next-meetup-talk": "My role as a developer",
+                    "next-meetup-speaker": "Henrique Moody"
+                },
+                "images": {
+                    "hoster-small": {
+                        "src": "img/hoster-logos/werkspot.svg",
+                        "alt": "Hosted by Werkspot"
+                    },
+                    "hoster-large": {
+                        "src": "img/hoster-logos/werkspot.svg",
+                        "alt": "Hosted by de Werkspot"
+                    },
+                    "hoster-next": {
+                        "src": "img/hoster-logos/usabilla-large.png",
+                        "alt": "Usabilla"
+                    },
+                    "speaker-face": {
+                        "src": "img/speakers/bob_den_otter.jpeg",
+                        "alt": "Speaker: Bob den Otter"
+                    }
+                }
+            };
+        </script>
+    </head>
+
+    <body>
+
+        <div class="reveal">
+
+            <div class="slides">
+                <section>
+                    <img src="img/logo-transparent-background.png" alt="AmsterdamPHP">
+                    <h4 id="date"></h4>
+
+                    <div class="two-columns" style="margin-top: 4em;">
+                        <div>
+                            <p>Hosted by</p>
+                            <img id="hoster-small" class="hoster" src="" alt=""">
+                        </div>
+
+                        <div>
+                            <ul class="pointless">
+                                <li>18h00 - 18h30: Welcome</li>
+                                <li>18h30 - 19h30: Dinner</li>
```

Comment:
Will the dinner be before of the talk again?

Reply:
@WyriHaximus Do you mind change it?

Reply:
Removed dinner